theme/static/js/hs_file_browser.js

```diff
@@ -742,32 +742,15 @@ function startDownload() {
 
     if (downloadList.length) {
         // Workaround for Firefox and IE
-        if (jQuery.browser.mozilla == true || !!navigator.userAgent.match(/Trident\/7\./)) {
```

Comment:
I talked to @Maurier about removing the chrome specific case.  Async downloads weren't working in chromeusing the downloadURI method.